docs/rfcs/XXX-pull-request-review.md

```diff
@@ -18,69 +18,126 @@ Peer review is also a critical part of the path to acceptance for pull requests
 
-![pending-review](https://user-images.githubusercontent.com/378023/46275946-9bd69680-c599-11e8-9889-66c35458286a.png)
+<img width="679" alt="xxx-pull-request-review_md_ ___github_github" src="https://user-images.githubusercontent.com/7910250/46391629-c8baa480-c692-11e8-8010-0cceb69394b1.png">
+
+
+### PullRequestDetailItem
+
+Each `PullRequestDetailItem` opened on a pull request displays the full, multi-file diff associated with the pull request. Review comments are shown within the diff. See ["Comment decorations"](#comment-decorations) for description of review comments.
+
+![screen shot 2018-10-03 at 1 50 18 pm](https://user-images.githubusercontent.com/7910250/46391711-1df6b600-c693-11e8-87f3-ad4cdbe8ebd8.png)
+
+TODO: update mock to have "Start review button" and "Add single comment"
+
+Diffs are editable ONLY if the pull request branch is checked out and the local branch history has not diverged from the remote branch history. Otherwise diffs are not editable.
+
+A panel at the bottom of the pane offers various options for sorting and filtering the diff. It also has a "Review Changes" button.
+
+#### Sort Options
+
+<img width="731" alt="slack_-_github" src="https://user-images.githubusercontent.com/7910250/46392358-f6551d00-c695-11e8-8ed4-c7aa95044b06.png">
+
+The default view is sorted by files. This is akin to the "Files changed" tab on dotcom. It displays the diff for all changed files in the PR.
+
+Sorting by reviews is akin to the review summaries that appear on the "Conversation" tab on dotcom. The comments are displayed grouped by review along with some context lines.
+
+![screen shot 2018-10-03 at 1 50 08 pm](https://user-images.githubusercontent.com/7910250/46394598-6ebfdc00-c69e-11e8-84eb-39ccbcccf736.png)
+
+TODO: include show multiple reviews stacked
+
+Sorting by commits is akin to the "Commits" tab on dotcom. A list of commits is displayed in chronological order, oldest commit on top. Clicking a commit expands the diff contents below. If there is a commit message body this is displayed as well.
+
+TODO: include commit sort mockup
+
+A banner at the bottom of the pane offers navigation to individual files within the diff and to individual review comments, allows each review to be hidden or shown with a filter control, and shows a progress bar that counts "resolved" review comments. The banner remains visible as you scroll the pane.
+
+#### Filter Options
+
+<img width="731" alt="slack_-_github" src="https://user-images.githubusercontent.com/7910250/46392373-03720c00-c696-11e8-9a1b-fe6bc6238769.png">
+
+The default is to show all files, all authors, and unresolved comments.
+
+Filtering based on file type limits the diff view to displaying only that file type.
+
+TODO: Consider adding a "Find" input field that allows us to filter based on search term (which could be a file name, an author, a variable name, etc). Probably out of scope for this RFC.
+
+Clicking an author's avatar displays only their review information.
```

Comment:
Good point. Maybe we can mark reviews as "Resolved"?

Reply:
Oops, clicked submit prematurely... I'm not sure if marking "Resolved" would necessarily address the use case, but it could be a way of filtering down the information presented to reduce noise. 

Agreed that the extra complexity might not be worth it. At least until we have a really solid use-case that applies to a lot of people.